cves/CVE-2015-0220.yml

```diff
@@ -173,8 +205,16 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: |
+    The subsystem name comes from the path of the file. It appears that django
+    uses a high level folder called django for their project, then uses
+    multiple high-level subsystems to separate where the function is coming
+    from. Within those subsystems are more subsystems with greater details
+    for specifics of the Django language. Since this one was a http issue, the
```

Comment:
"the Django language" I do believe it is a framework, and claiming it to be a language may confuse some readers, consider updating.

Reply:
"Since this one was a http issue" - consider replacing "a" with "an" and being more specific than using the word "this one".

Reply:
Additionally remove the word "the" it appears to be an extra word.